third_party/ijar/BUILD

```diff
@@ -30,6 +30,7 @@ cc_library(
         "//src:__subpackages__",
         "//third_party/ijar:__subpackages__",
         "//tools/test:__pkg__",
+        "@local_java_tools//:__pkg__",
```

Comment:
I'm not sure this is enough. It's enough for the tests to pass. The remote repo names in the workspace suffix are "remote_java_tools_linux", "remote_java_tools_windows" and "remote_java_tools_darwin". Can you please add them as well? Also add a TODO to migrate this target out of bazel_tools.